### PR TITLE
Bug/defra ffc 142028

### PR DESCRIPTION
**Pull request by liamwilson-kainos**

### CODE REVIEW

**Review by govTechSatish — DISMISSED** [review]

File: app/register-your-interest/messaging/process-register-your-interest-request.js [hunk 1]
```diff
@@ -8,9 +8,9 @@ const processRegisterYourInterestRequest = async (request) => {
     throw new Error(req.error)
   }
   const { sbi, crn, email } = req.value
-  const eligible = await checkEligibility(sbi, crn)
+  const eligible = await checkEligibility(sbi, crn, email)
```
Comment (AbidemiAdio) [inline app/register-your-interest/messaging/process-register-your-interest-request.js:11]: how about `{ waiting_updated_at, access_granted } = checkEligibility(sbi, crn, email)`

File: app/register-your-interest/messaging/process-register-your-interest-request.js [hunk 2]
```diff
@@ -8,9 +8,9 @@ const processRegisterYourInterestRequest = async (request) => {
...
   if (eligible) {
-    await processEligible(sbi, crn, email)
+    await processEligible(sbi, crn, email, eligible.waiting_updated_at, eligible.access_granted)
```
Comment (AbidemiAdio) [inline app/register-your-interest/messaging/process-register-your-interest-request.js:13]: how about  processEligible(sbi, crn, email, waiting_updated_at, access_granted)

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-eligibility",
-  "version": "0.16.0",
+  "version": "0.16.1",
```
Comment (AbidemiAdio) [inline package.json:3]: note: there may be version number conflict with previous PR

File: test/unit/app/register-your-interest/messaging/process-register-your-interest-request.test.js [hunk 4]
```diff
@@ -41,17 +41,20 @@ describe('Process register your interest message', () => {
...
 
     await processRegisterYourInterestRequest(request)
 
     expect(logSpy).toHaveBeenCalledWith(`Processing register your interest request: ${JSON.stringify(request)}`)
     expect(processEligible).toHaveBeenCalledWith(
       request.sbi,
       request.crn,
-      request.email
+      request.email,
+      '2022-12-29 11:35:00',
```
Comment (AbidemiAdio) [inline test/unit/app/register-your-interest/messaging/process-register-your-interest-request.test.js:56]: value as variable name may improve readability

File: app/register-your-interest/messaging/process-register-your-interest-request.js (hunk 1 above)
Comment (liamwilson-kainos) [reply app/register-your-interest/messaging/process-register-your-interest-request.js:11]: Make sense 👌

File: package.json (hunk 3 above)
Comment (liamwilson-kainos) [reply package.json:3]: sure, will bump mine to`0.16.2`

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**